app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -51,6 +57,38 @@ class AddDeadlineActivity : AppCompatActivity() {
         findViewById<CheckBox>(R.id.radio_notification_3d).text =
             getString(R.string.notify_before, "3 days")
         selectedDate = clockService.now().truncatedTo(ChronoUnit.MINUTES)
+        updateDisplayedDateAndTime()
+        KeyboardVisibilityEvent.setEventListener(this, object : KeyboardVisibilityEventListener {
+            override fun onVisibilityChanged(isOpen: Boolean) {
```

Comment:
During this sprint clearly not, maybe in a future one but this would probably take a lot of time and time is precious given all what remains to be done...